src/GraphQL/Types/DateGraphType.cs

```diff
@@ -17,13 +18,27 @@ public override object Serialize(object value)
 
             if (date is DateTime dateTime)
             {
-                return dateTime.ToString("yyyy-MM-dd");
+                return dateTime.ToString("yyyy-MM-dd", CultureInfo.InvariantCulture);
             }
 
             return null;
         }
 
-        public override object ParseValue(object value) => ValueConverter.ConvertTo(value, typeof(DateTime));
+        public override object ParseValue(object value)
+        {
+            if (value is DateTime dateTime)
+            {
+                return dateTime;
+            }
+
+            var valueAsString = (string)value;
```

Comment:
consider to change this to `if (value is string valueAsString)`

Reply:
yep. agreed. done